test/blackbox-tests/test-cases/merlin/merlin-tests.t/run.t

```diff
@@ -9,14 +9,12 @@ CRAM sanitization
   X
   ((STDLIB /OCAMLC_WHERE)
    (EXCLUDE_QUERY_DIR)
-   (B lib/bytes)
```

Comment:
This is because `ocamlfind` installs a dummy `META` file in an empty directory `$SITELIB/bytes`
```bash
nojebar@VANISHINGCYCLE:~/.opam/4.11/lib/bytes$ cat META
name="bytes"
version="[distributed with OCaml 4.02 or above]"
description="dummy backward-compatibility package for mutable strings"
requires=""
```
this file is no longer "seen" by `dune` with the change in this PR.